src/main/java/networkbook/logic/commands/EditCommand.java

```diff
@@ -148,8 +149,8 @@ public String toString() {
     public static class EditPersonDescriptor {
         private Name name;
         private Phone phone;
-        private UniqueList<Email> emails;
-        private UniqueList<Link> links;
+        private Optional<UniqueList<Email>> emails = Optional.empty();
+        private Optional<UniqueList<Link>> links = Optional.empty();
```

Comment:
Referring to the discussion [here](https://stackoverflow.com/questions/29033518/is-it-a-good-practice-to-use-optional-as-an-attribute-in-a-class), using `Optional` as a field type is not the recommended practice. 

This is not an urgent issue. When you are free, can you change these fields to be of type `UniqueList<>`, and use the `Optional` constructor in the getter method instead? You'll also need to change the setter methods as well, can refer to my implementation of `tags` if you want. Thanks!

Reply:
Noted wth thanks!